### PR TITLE
Issue Lists as Pull Request Comments

### LINKED ISSUE #1031: Code review checklist
I would love to be able to configure (on a per-repo basis perhaps) a code reviewers checklist that can be used to guide the review process.  I don't know if something simple could be done with an embedded [GFM tasklist](https://github.com/blog/1375%0A-task-lists-in-gfm-issues-pulls-comments) or if this would need to be additional functionality and state tracked in reviewninja itself.  I would also see this being useful as another pre-req for enabling merging (N ninja stars + no open issues + review checklist complete)

### PR DESCRIPTION
**Pull request by casche**
under 🚧 

closes #1031

We're going to allow lists of issues inside Pull Request comments to create RN code review problems. This will allow [Pull Request templates](https://github.com/reviewninja/review.ninja/pull/1067) to create a list of issues that contributors must solve before changing the status.

e.g. 

```
!fix
- [ ] one
- [ ] two
- [ ] three
=> negative status
```
```
!fix
- [x] one
- [x] two
- [x] three
=> positive status
```

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/review.ninja/pull/1068" target="_blank">ReviewNinja

File: src/server/services/flags.js [hunk 1]
```diff
@@ -8,13 +8,16 @@ module.exports = {
 
     review: function(comments) {
 
+        // TODO: enhance to parse markdown lists e.g. [] -> [ ]
```
Comment (casche) [inline src/server/services/flags.js]: !fix remove this comment

Comment (casche) [inline src/server/services/flags.js]: !fix remove the TODO comment

Comment (casche) [inline src/server/services/flags.js]: !fixed

Comment (casche) [inline src/server/services/flags.js]: !fix something

**Pull request merged**